pipeline/src/test_util.rs

```diff
@@ -112,6 +112,12 @@ pub fn gen_estark_proof(file_name: &str, inputs: Vec<GoldilocksField>) {
     pipeline.verify(&proof, &[publics]).unwrap();
 }
 
+/// A subset of BackendType, but also available without the halo2 feature.
+pub enum Halo2Backend {
```

Comment:
Why not call it something without Halo2 and then also use it with the other backends?

Reply:
True!